Tests/UnitTests/Misc/TimingUtilTests.swift

```diff
@@ -175,22 +187,32 @@ class TimingUtilAsyncTests: TestCase {
             level: level
         )
     }
+
+    private static func asyncMethod() async {}
```

Comment:
Hmm is this used for something?

Reply:
Simply to verify that the calls to `measure` in those tests take an `async` closure. They do `await Self.asyncMethod` to verify that.